TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -86,7 +109,7 @@ def initialize(self):
     # See cleanCatalogContents method: this proxy will be used ALSO when the file catalog used
     # is the DIRAC File Catalog (DFC).
     # This is possible because of unset of the "UseServerCertificate" option
-    self.am_setOption('shifterProxy', 'DataManager')
+    self.shifterProxy = self.am_getOption('shifterProxy', None)
```

Comment:
You should update the wiki for this one.

Reply:
Done: https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v6r20#transformationsystem-v6r20p6